modules/images-other-jenkins-oauth-auth.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins.adoc
+
+[id="images-other-jenkins-oauth-auth_{context}"]
+= {product-title} OAuth authentication
+
+OAuth authentication is activated by configuring options on the *Configure
+Global Security* panel in the Jenkins UI, or by setting the
+`OPENSHIFT_ENABLE_OAUTH` environment variable on the Jenkins *Deployment
+configuration* to anything other than `false`. This activates the {product-title}
+Login plug-in, which retrieves the configuration information from Pod data or by
+interacting with the {product-title} API server.
+
+Valid credentials are controlled by the {product-title} identity provider.
+ifdef::openshift-enterprise,openshift-origin,openshift-dedicated[]
+For example, if `Allow All` is the default identity provider, you can provide
+any non-empty string for both the user name and password.
+endif::[]
+
+Jenkins supports both browser and non-browser access.
+
+Valid users are automatically added to the Jenkins authorization matrix at log
+in, where {product-title} `Roles` dictate the specific Jenkins permissions the
+users have.
+
+Users with the `admin` role have the traditional Jenkins administrative
+user permissions. Users with the `edit` or `view` role have progressively
+fewer permissions.
+
```

Comment:
The new configurable role mapping feature should be dropped in somewhere around here

See https://github.com/openshift/jenkins-openshift-login-plugin#openshift-role-to-jenkins-permission-mapping